src/utils/Fuse.js

```diff
@@ -0,0 +1,27 @@
+const Fuse = require('fuse.js')
+
+let fuse, commandList
```

Comment:
why this?

Reply:
Because `commandList` only can be filled when bot full started, so I create this variables to when user uses a command by first time, the bot fills it, `fuse` must be a variable too, because it depends `commandList`